index.html

```diff
@@ -61,21 +65,17 @@
 							box-shadow: 0px 3px 2px rgba(0, 0, 0, 0.2);
 						}
 
-						paper-tabs::shadow #selectionBar {
+						paper-tabs #selectionBar {
```

Comment:
Ah, brilliant. I tried that before but I think I may have seen the docs for the newer version online while having an old and incompatible version locally. Looks much nicer now.